draft-yasskin-http-origin-signed-responses.md

```diff
@@ -482,12 +484,12 @@ there are no non-significant response header fields in the exchange.
             certificate_request_context MUST be empty, and a non-empty value
             MUST cause the parse to fail.
       1. Let `main-certificate` be the first certificate in `certificate-chain`.
-      1. If the SHA-256 hash of `main-certificate`'s `cert_data` is not equal to
-         `certSha256`, return "invalid". Note that this intentionally differs
-         from TLS 1.3, which signs the entire certificate chain in its
-         Certificate Verify (Section 4.4.3 of {{?I-D.ietf-tls-tls13}}), in order
-         to allow updating the stapled OCSP response without updating signatures
-         at the same time.
+      1. If the SHA-256 hash of `certificate-chain` is not equal to
+         `certSha256`, return "invalid".  Note that this requires that
+         signatures be updated at the same time as the certificate OCSP
+         responses they cover.  Intermediates can use distinct `certUrl`s for
```

Comment:
s/can/SHOULD/ maybe? It seems, in general, that `certUrl`s should change whenever their contents change, otherwise old signed extensions would fail to validate.